configs/opencv-ovms/gst_capi/pipelines/capi_yolov5/main.cpp

```diff
@@ -520,17 +520,19 @@ class Yolov5 : public ObjectDetectionInterface {
             const float xc = (x1-x0)/2.0f + x0;
             const float yc = (y1-y0)/2.0f + y0;
 
-            const float scaled_x0 = std::clamp(xc-obj.width*scaler_w / 2.0f, 0.f, static_cast<float>(_window_width));
+            const float scaled_x0 = std::clamp(xc-obj.width*scaler_w, 0.f, static_cast<float>(_window_width));
```

Comment:
this is mathematically wrong why remove /2 ?

Reply:
I can show you again how it appears on the screen.